pkg/sidecar/config.go

```diff
@@ -30,6 +30,12 @@ type Config struct {
 	SvidBundleFileNameDeprecated       string `hcl:"svidBundleFileName"`
 	RenewSignal                        string `hcl:"renew_signal"`
 	RenewSignalDeprecated              string `hcl:"renewSignal"`
+
+	// JWT configuration
+	JwtAudience            string `hcl:"audience"`
+	JSONFilename           string `hcl:"json_filename"`
+	JSONFilenameDeprecated string `hcl:"jsonFilename"`
```

Comment:
We dont need the deprecated stuff. Can you remove?

Reply:
Sure, makes sense